xrpl/asyncio/clients/json_rpc_base.py

```diff
@@ -38,7 +38,7 @@ async def request_impl(self: JsonRpcBase, request: Request) -> Response:
 
         :meta private:
         """
-        async with AsyncClient() as http_client:
+        async with AsyncClient(timeout=10.0) as http_client:
```

Comment:
default timeout is 5.0s, some integration tests were failing intermittently because they timed out